src/main/java/com/ncedu/cheetahtest/service/cloud/AmazonClientService.java

```diff
@@ -0,0 +1,105 @@
+    private String bucketName;
+    @Value("${cloud.aws.credentials.accessKey}")
+    private String accessKey;
+    @Value("${cloud.aws.credentials.secretKey}")
+    private String secretKey;
+
+    @PostConstruct
+    private void initializeAmazon() {
+        AWSCredentials credentials = new BasicAWSCredentials(this.accessKey, this.secretKey);
+        this.s3client = new AmazonS3Client(credentials);
+    }
+
+    // its maximum permitted size of 20971520 bytes.
+    public String uploadUserPhoto(MultipartFile multipartFile, int id) {
+        String fileUrl = "";
+        try {
+            File file = convertMultiPartToFile(multipartFile);
+            String fileName = generateFileName(multipartFile);
+            fileUrl = "https://"+bucketName+endpointUrl+"/userPhoto/"+fileName;
+            uploadFileTos3bucket("userPhoto/"+fileName, file);
+            file.delete();
+            userService.setUserPhotoUrl(id, fileUrl);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return fileUrl;
+    }
+
+    // its maximum permitted size of 20971520 bytes.
+    public String uploadScreenshot(File file) {
+        String fileUrl = "";
+        try {
+            String fileName = generateFileName(file);
+            fileUrl = "https://"+bucketName+endpointUrl+"/screenshots/"+fileName;
+            uploadFileTos3bucket("screenshots/"+fileName, file);
+            file.delete();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return fileUrl;
+    }
+
+    public File convertMultiPartToFile(MultipartFile file) throws IOException { //TODO return to private
+        File convFile = new File(file.getOriginalFilename());
+        FileOutputStream fos = new FileOutputStream(convFile);
+        fos.write(file.getBytes());
+        fos.close();
+        return convFile;
+    }
+
+    private String generateFileName(MultipartFile multiPart) {
+        return new Date().getTime() + "-" + multiPart.getOriginalFilename().replace(" ", "_");
+    }
+
+    private String generateFileName(File file) {
+        return new Date().getTime() + "-" + file.getName().replace(" ", "_");
+    }
+
+    private void uploadFileTos3bucket(String fileName, File file) {
+        s3client.putObject(new PutObjectRequest(bucketName, fileName, file)
+                .withCannedAcl(CannedAccessControlList.PublicRead));
+    }
+
+    public String deleteUserPhotoFromS3Bucket(String fileUrl) {
+        String fileName = fileUrl.substring(fileUrl.lastIndexOf("/") + 1);
+        s3client.deleteObject(new DeleteObjectRequest(bucketName, "userPhoto/"+fileName));
+        return "Successfully deleted";
+    }
+
+    public String deleteScreenshotFromS3Bucket(String fileUrl) {
+        String fileName = fileUrl.substring(fileUrl.lastIndexOf("/") + 1);
+        s3client.deleteObject(new DeleteObjectRequest(bucketName, "screenshots/"+fileName));
+        return "Successfully deleted";
+    }
```

Comment:
This methods is almost the same. Please, move same logic to separate method. Better make this methods void and  log "Successfully deleted" instead of returning it.